### PR TITLE
ref(lint): Fix Typos

### PR DESCRIPTION
**Pull request by mgaeta**
Collected a few more typo fixes. Each commit is a different class of fix.

### CODE REVIEW

**Review by maxiuyuan — APPROVED** [review]

**Comment by maxiuyuan** [comment]
lgtm, hopefully there isn't any typos that slip by me

File: src/sentry/receivers/onboarding.py [hunk 1]
```diff
@@ -254,7 +254,7 @@ def record_release_received(project, event, **kwargs):
             user = Organization.objects.get(id=project.organization_id).get_default_owner()
         except IndexError:
             logging.getLogger("sentry").warning(
-                "Cannot record release recieved for organization (%s) due to missing owners",
+                "Cannot record release received for organization (%s) due to missing owners",
```
Comment (JoshFerge) [inline src/sentry/receivers/onboarding.py:257]: have we confirmed nowhere is grepping on this exact log string?

File: src/sentry/integrations/issues.py [hunk 2]
```diff
@@ -218,7 +218,7 @@ def get_repository_choices(self, group, **kwargs):
         try:
             repos = self.get_repositories()
         except ApiError:
-            raise IntegrationError("Unable to retrive repositories. Please try again later.")
+            raise IntegrationError("Unable to retrieve repositories. Please try again later.")
```
Comment (JoshFerge) [inline src/sentry/integrations/issues.py:221]: also this one: downstream is anything depending on this exact error message?

**Comment by mgaeta** [comment]
We're not relying on this string anywhere and I can't find anyone else who might be.
Screen Shot 2021-07-19 at 9 31 40 AM

**Pull request merged**